### PR TITLE
Ms collector alignments

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-3917

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_14_0.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+##### New: Microsoft Defender for Endpoint Event Collector
+Microsoft 365 Defender event collector integration for Cortex XSIAM.
```
Comment (Shellyber) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_14_0.md:4]: Need to change the name here also, no?

**Review by Shellyber — COMMENTED** [review]
Don't you want to change the package name and files as well? not just move them to the right pack

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_14_0.md (hunk 1 above)
Comment (ShahafBenYakir) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_14_0.md:4]:
```suggestion
Microsoft Defender for Endpoint Event Collector integration for Cortex XSIAM.
```

**Comment by ShahafBenYakir** [comment]
@Shellyber I thought about it but i fear it will create more confusion as there will be discrepancy between the names Ids and file names

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_14_0.md (hunk 1 above)
Comment (ShahafBenYakir) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_14_0.md]: Good catch

**Review by Shellyber — APPROVED** [review]
Looks good.
I think that we need to change the names anyway, it is very confusing now, the pack and the integration files are different, but I leave it for you to decide. 

Notice that you have some errors in validation and in the .guardrails.

File: Packs/Microsoft365Defender/ReleaseNotes/4_4_0.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Microsoft 365 Defender Event Collector
+Microsoft 365 Defender event collector was moved to the **Microsoft Defender for Endpoint** pack and was renamed 
+to **Microsoft Defender for Endpoint Event Collector**.
```
Comment (dantavori) [inline Packs/Microsoft365Defender/ReleaseNotes/4_4_0.md:5]:
```suggestion
Microsoft 365 Defender event collector was moved to the ***Microsoft Defender for Endpoint*** pack and was renamed to ***Microsoft Defender for Endpoint Event Collector***.
```

**Review by dantavori — APPROVED** [review]

**Pull request merged**